algorithms/Lesson2_Arrays/ex_1.rb

```diff
@@ -0,0 +1,8 @@
+def solution(array)
+  counter = Hash.new 0
+  array.each.with_index do |num, _i|
```

Comment:
if you're not using an index, you should use `each` instead of `each_with_index`